helm/overstroomik/templates/ingress.yaml

```diff
@@ -11,13 +11,12 @@ metadata:
   {{- end }}
 spec:
   rules:
-  - http:
-      paths:
-      {{- range .Values.ingress.services}}
-      - path: {{ .path }}
-        backend:
-          serviceName: {{ .name }}
-          servicePort: {{ .port }}
-      {{- end }}
+    - http:
```

Comment:
I would expect - host below rules. host is added to the values.yaml, but it is not used in the ingress template